diff -aur xine-ui-0.99.11.orig/src/xitk/help.c xine-ui-0.99.11/src/xitk/help.c
--- xine-ui-0.99.11.orig/src/xitk/help.c	2019-05-28 12:20:58.000000000 +0200
+++ xine-ui-0.99.11/src/xitk/help.c	2019-05-28 17:20:38.560009930 +0200
@@ -180,7 +180,7 @@
 	  help_add_section(locale_readme, lang->doc_encoding, order_num, section_name);
 	}
 	else {
-	  help_add_section(default_readme, "ISO-8859-1", order_num, section_name);
+	  help_add_section(default_readme, "UTF-8", order_num, section_name);
 	}
       }
     }
